gc-worker/server/server.go

```diff
@@ -0,0 +1,279 @@
+package server
+
+import (
+	"context"
+	"flag"
+	"fmt"
+	"gc-worker/server/config"
+	"strings"
+	"sync"
+	"sync/atomic"
+	"time"
+
+	"github.com/docker/go-units"
+	"github.com/pingcap/errors"
+	"github.com/pingcap/log"
+	pd "github.com/tikv/pd/client"
+	"go.etcd.io/etcd/clientv3"
+	"go.etcd.io/etcd/clientv3/concurrency"
+	"go.uber.org/zap"
+	"google.golang.org/grpc"
+)
+
+const (
+	etcdTimeout = time.Second * 3
+	// gcWorkerRootPath for all gcworker servers.
+	gcWorkerRootPath      = "/gc-worker/selection"
+	gcWorkerElectionVal   = "local"
+	maxMsgSize            = int(128 * units.MiB)
+	updateGCPointInterval = 10 * 1000 * 1000 * 1000 // 10 minutes
```

Comment:
Can those parameters be configured through the configuration file?

Reply:
sure, I would like to do the configuration in next PR as it may make this PR too big. And i think etcd path should be hard coded?

Reply:
has added the these to configure files